test/tests/meshgenerators/block_deletion_generator/delete_interior_parents.i

```diff
@@ -0,0 +1,76 @@
+[Mesh]
+  [twoquad]
+    type = GeneratedMeshGenerator
+    dim = 2
+    nx = 3
+    ny = 4
+    xmax = 2
+  []
+  [subdomain]
+    type = SubdomainBoundingBoxGenerator
+    input = twoquad
+    block_id = 1
+    bottom_left = '1 0 0'
+    top_right = '2 1 0'
+  []
+  [sideset]
+    type = SideSetsBetweenSubdomainsGenerator
+    input = subdomain
+    primary_block = 0
+    paired_block = 1
+    new_boundary = bar
+  []
+  [edge_elem]
+    type = LowerDBlockFromSidesetGenerator
+    input = sideset
+    sidesets = bar
+    new_block_id = 2
+  []
```

Comment:
duplicate those please with foo for the sideset from 1 to 0 ((instead of 0 to 1)

I want to see if those lowerD elements survive as expected